server/containers/docker/Dockerfile

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright 2014-2016 CyberVision, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM ubuntu:16.04
+MAINTAINER KAA IoT PLATFORM <www.kaaproject.org>
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && apt-get -yqq install \
+		software-properties-common \
+		python-software-properties \
+		ca-certificates \
+		netcat \
+		nano \
+		vim  \
+		curl \
+		net-tools \
+		mc \
```

Comment:
please make sure that all this tools are necessary. No need to waste disk space :-)